### PR TITLE
Add changelog

### PR DESCRIPTION
**Pull request by Hanzei**
This is a first draft of the changelog. Please be free to add more notable changes and correct my spelling mistakes :smile:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+### Added
+- Polls are posted as a bot user
+- You can use single and double quotation instead of backticks
+- We now verify the Mattermost token to make sure requests are valid
+- The ip address Matterpoll should listen for request can now we configured via `config.json`
+- You can choose the config file via `-c` option
+- Go 1.6 to 1.9 are now supported
+
+### Changed
+- Removed `-p` option. Choosing the port is now done via `config.json`
```
Comment (kaakaa) [inline CHANGELOG.md]: I think that removing `-p` option is breaking change.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  - `listen` defines the ip address and port Matterpoll will listen for requests. Default is `localhost:8505`
+  - `token` is the mattermost token. You should copy it from Integration > Slash Commands at your Mattermost server.
+
+### Added
+- Polls are posted as a bot user
+- You can use single and double quotation instead of backticks
+- We now verify the Mattermost token to make sure requests are valid
+- The ip address Matterpoll should listen for request can now we configured via `config.json`
+- You can choose the config file via `-c` option
+- Go 1.6 to 1.9 are now supported
```
[the remark this answers is not in the document]
Comment (Hanzei) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+### Fixed
+- Removed dependency of glide
+- Mattermost warning about empty message
+
+## 0.0.2 – 2017-04-19
+### Added
+- Ci builds via travis
+
+## Changed
```
[the remark this answers is not in the document]
Comment (Hanzei) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md (hunk 1 above)
Comment (Hanzei) [reply CHANGELOG.md]: :+1:

**Pull request merged**